os-ovs.te

```diff
@@ -28,6 +29,7 @@ gen_require(`
 	class netlink_audit_socket { create nlmsg_relay read write };
 	class capability audit_write;
 	class packet_socket create_socket_perms;
+        class unix_stream_socket { read write connectto };
```

Comment:
Not sure if 'connectto' is needed, it doesn't seem used?

Reply:
unix_stream_socket / connectto is on line 54, actually - it was not explicitly listed before, but should have been:

allow svirt_t openvswitch_t:unix_stream_socket connectto;